Math/Convert.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+
+
+// Yes i really did make a new file for a single function, i want my code to look clean.
```

Comment:
No need for this comment here :) This is stuff that can be put in a PR description